src/lamp_py/bus_performance_manager/gtfs_rt_ingestion.py

```diff
@@ -0,0 +1,115 @@
+from datetime import date
+from typing import List
+
+import polars as pl
+
+from lamp_py.bus_performance_manager.gtfs_utils import (
+    bus_routes_for_service_date,
+)
+
+
+def generate_gtfs_rt_events(
+    service_date: date, gtfs_rt_files: List[str]
+) -> pl.DataFrame:
+    """
+    generate a polars dataframe for bus vehicle events from gtfs realtime
+    vehicle position files for a given service date
+    """
+    bus_routes = bus_routes_for_service_date(service_date)
+
+    # build a dataframe of every gtfs record
+    # * scan all of the gtfs realtime files
+    # * only pull out bus records and records for the service date with current status fields
+    # * rename / convert columns as appropriate
+    # * convert
+    # * sort by vehicle id and timestamp
+    # * keep only the first record for a given trip / stop / status
+    vehicle_positions = (
+        pl.scan_parquet(gtfs_rt_files)
+        .filter(
+            (pl.col("vehicle.trip.route_id").is_in(bus_routes))
+            & (
+                pl.col("vehicle.trip.start_date")
+                == service_date.strftime("%Y%m%d")
+            )
+            & pl.col("vehicle.current_status").is_not_null()
+            & pl.col("vehicle.stop_id").is_not_null()
+            & pl.col("vehicle.trip.trip_id").is_not_null()
+            & pl.col("vehicle.vehicle.id").is_not_null()
+            & pl.col("vehicle.timestamp").is_not_null()
+        )
+        .select(
+            pl.col("vehicle.trip.route_id").cast(pl.String).alias("route_id"),
```

Comment:
I think we're missing a few columns needing to be pulled, as defined by the bus requirements notion Doc. 

such as:
stop_sequence
vehicle_label

Reply:
i went through the doc and pulled all the additional gtfs_rt sourced fields. it was just these two.